edgelet/edgelet-core/src/authorization.rs

```diff
@@ -16,30 +14,44 @@ pub enum Policy {
     Module(&'static str),
 }
 
-pub struct Authorization<M> {
-    runtime: M,
+#[derive(Clone, Debug, PartialEq)]
+pub enum AuthId {
+    None,
+    Any,
+    Value(String),
+}
+
+impl fmt::Display for AuthId {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            AuthId::None => write!(f, "none"),
+            AuthId::Any => write!(f, "any"),
```

Comment:
My c# brain was confused here because Any came before the Value, so I was wondering how Value ever got called :)